test/e2e/test/elasticsearch/steps_mutation.go

```diff
@@ -77,9 +75,9 @@ func (b Builder) MutationTestSteps(k *test.K8sClient) test.StepList {
 			},
 			test.Step{
 				Name: "Data added initially should still be present",
-				Test: func(t *testing.T) {
-					require.NoError(t, dataIntegrityCheck.Verify())
-				},
+				Test: test.Eventually(func() error { // nolint
+					return dataIntegrityCheck.Verify()
```

Comment:
This should be `test.Eventually(dataIntegrityCheck.Verify)` but that does not work and panics which I don't fully understand yet